packages/lib-classifier/src/components/Classifier/components/withKeyZoom/withKeyZoom.js

```diff
@@ -2,7 +2,7 @@ import { MobXProviderContext } from 'mobx-react'
 import React, { Component, forwardRef, useContext } from 'react'
 
 function withKeyZoom (WrappedComponent) {
-  const ALLOWED_TAGS = ['svg', 'button', 'rect']
+  const ALLOWED_TAGS = ['svg', 'button', 'g', 'rect']
```

Comment:
This should update the specs. I would've expected a failure since there are explicit tests for  when the target is a certain element or not

Reply:
Sorry, I missed that.